packages/esm-appointments-app/translations/en.json

```diff
@@ -62,6 +62,7 @@
   "createNewAppointment": "Create new appointment",
   "date": "Date",
   "date&Time": "Date & time",
+  "dateAppointmentIssuedCannotBeAfterAppointmentDate": "Date appointment issued cannot be after the appointment date",
```

Comment:
This is likely okay, looks like it's coming from a previous change where the hook to update translations must not have been run.